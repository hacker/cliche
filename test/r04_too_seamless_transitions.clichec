#include <iostream>
int main() {
int a=1,b=2;
<%output>
The transition like this - <% a %><% b %> should not glue 'ab' in the source together.
Really.
</%output>
}

