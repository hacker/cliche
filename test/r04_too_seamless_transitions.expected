
The transition like this - 12 should not glue 'ab' in the source together.
Really.
